.github/CODEOWNERS

```diff
@@ -1 +1,13 @@
-* @hashicorp/nomad-eng
+# Each line is a file pattern followed by one or more owners.
+# More on CODEOWNERS files: https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
+
+# Default owner
+* @hashicorp/team-ip-compliance @hashicorp/nomad-eng
```

Comment:
Github says `hashicorp/team-ip-compliance` doesn't exist? Is there a typo here?

Reply:
No typo, we are waiting on getting membership request sorted out. Will message on slack with the details.

Reply:
The permissions issue is now resolved. We are ready to merge this one.